query.css

```diff
@@ -0,0 +1,356 @@
+  }
+
+  .desktop-nav ul li a {
+    color: #344563;
+  }
+
+  .hero {
+    background-image: url(./assets/desktop/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    justify-content: center;
+    align-items: center;
+    border-bottom-left-radius: 100px;
+    height: 750px;
+  }
+
+  .author {
+    margin-left: 200px;
+    padding-top: 200px;
+    width: 30%;
+    justify-content: center;
+    letter-spacing: 0.37px;
+  }
+
+  .bio {
+    margin-left: 200px;
+    width: 60%;
+    height: auto;
+    letter-spacing: 0.37px;
+  }
+
+  .connect_button {
+    margin-left: 200px;
+    font-size: 20px;
+  }
+
+  .social ol {
+    margin-left: 200px;
+  }
+
+  .desktop-card,
+  .card {
+    background-color: white;
+    display: flex;
+    height: 400px;
+    width: 85%;
+    padding: 24px;
+
+    /* margin-bottom: 70px; */
+    border-radius: 24px;
+  }
+
+  .desktop-preview,
+  .preview {
+    grid-column: 1/-1;
+    justify-self: center;
+    align-self: center;
+  }
+
+  .desktop-preview img,
+  .preview img {
+    margin-top: 10px;
+    width: 100%;
+    height: 350px;
+  }
+
+  .desktop-details,
+  .details {
+    grid-column: 1/-1;
+    align-self: center;
+    margin-left: 50px;
+    padding: 10px;
+  }
+
+  .desktop-title,
+  .title {
+    margin: 10px 0;
+    color: #172b4d;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+  }
+
+  .desktop-info,
+  .info {
+    display: flex;
+    overflow: hidden;
+    color: #7a869a;
+  }
+
+  .desktop-info_item,
+  .info_item {
+    margin-right: 15px;
+    list-style-position: inside;
+  }
+
+  .desktop-info_item:first-child,
+  .info_item:first-child {
+    font-weight: bold;
+    list-style: none;
+    color: #344563;
+  }
+
+  .desktop-info_item:last-child,
+  .info_item:last-child {
+    margin-right: 0;
+  }
+
+  .desktop-details_content,
+  .details_content {
+    width: 80%;
+    margin: 10px 0;
+    padding: 10px 0;
+    color: #344563;
+  }
+
+  .desktop-catagories,
+  .catagories {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 20px 0;
+  }
+
+  .catagories .special {
+    display: flex;
+  }
+
+  .desktop-catagories ul li,
+  .catagories ul li {
+    padding: 8px;
+    color: #6070ff;
+    border-radius: 8px;
+    margin-right: 8px;
+    background: #ebebff;
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .desktop-default_button,
+  .default_button {
+    cursor: pointer;
+    font-size: 18px;
+    margin-top: 20px;
+    color: #396df2;
+    padding: 10px 20px;
+    background: #fff;
+    border-radius: 10px;
+    border: 1px solid #396df2;
+  }
+
+  .desktop-default_button:hover,
+  .default_button:hover {
+    color: #fff;
+    background: #396df2;
+  }
+
+  #order-4 {
+    order: 1;
+    flex-direction: row-reverse;
+  }
+
+  #order-2 {
+    order: 3;
+    flex-direction: row-reverse;
+  }
+
+  #order-1 {
+    order: 2;
+  }
+
+  #order-3 {
+    display: none;
+  }
+
+  /* end card */
+
+  /* about */
+  .about {
```

Comment:
:red_circle: Kindly notice the top-right border radius on this section. Please make sure you add the radius to match the Figma design

Your Design
![border](https://user-images.githubusercontent.com/26736582/200088635-280464fe-d0d0-4a19-b482-8cf4a29da58b.png)

Figma Design
![borderfigma](https://user-images.githubusercontent.com/26736582/200088655-cfc12ddd-8070-4cfc-b52b-6c619cd816fc.png)

Reply:
- [ ] Make sure this issue is addressed with the feedback from the previous reviewer about the border radius for the top right corner based on the Figma template design; please fix this issue